src/components/Fullscreen/Fullscreen.tsx

```diff
@@ -18,12 +18,15 @@ interface FullscreenProps {
     className?: string;
 }
 
-const fullscreenRoot = document.getElementById('fullscreen-root') ?? undefined;
-
-function Fullscreen({children, className}: FullscreenProps) {
+export function Fullscreen({children, className}: FullscreenProps) {
     const isFullscreen = useTypedSelector((state) => state.fullscreen);
     const dispatch = useTypedDispatch();
 
+    const fullscreenRoot = React.useMemo(
+        () => document.getElementById('fullscreen-root') ?? undefined,
+        [],
+    );
```

Comment:
In dev mode root sometimes was not found - it's undefined, fullscreen doesn't work. No it should always be present
<img width="800" alt="Screenshot 2025-09-03 at 09 23 41" src="https://github.com/user-attachments/assets/8994b5b2-2cc4-4f95-abe1-ac0c90ce57f4" />